src/components/AudioRecorderCompairUI/AudioRecorderCompairUI.js

```diff
@@ -31,6 +36,88 @@ export default class AudioRecorderCompair extends Component {
     });
   }
 
+  handleMic(){
+    if(!(this.pathname === 'practice')){
```

Comment:
instead of practice - this should be true or false
True - enabled audio pre-processing
False - disabled audio pre-processing

Reply:
done